packages/sources/src/components/card-list/card-list.stories.ts

```diff
@@ -0,0 +1,57 @@
+import { bindTemplate, StorybookParameters } from '../../stories-helpers';
+
+import { cardListArgsMapper, cardListArgTypes } from './card-list.args';
+import { CardList } from './card-list.models';
+
+export interface CardListParameters<TemplateFnReturnType> {
+  cardListTemplate: (cardListProperties: CardList) => TemplateFnReturnType;
+}
+
+export function storiesOfCards<TemplateFnReturnType>(
+  {
+    module: mainModule,
+    storiesOf,
+    readme
+  }: StorybookParameters,
+  {
+    cardListTemplate
+  }: CardListParameters<TemplateFnReturnType>
+) {
+  const template = bindTemplate(cardListArgsMapper, cardListTemplate);
+
+  const stories = storiesOf('Card List', mainModule)
+    .addParameters({
+      docs: {
+        page: readme
+      },
+      argTypes: cardListArgTypes,
+      args: {
+        cards: [
+          {
+            label: 'Omgevingsplan Nieuwegein',
+            content: 'Gemeente Nieuwegein lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+          },
+          {
+            label: 'Brouwersmolen',
+            content: 'Brouwersmolen eget iaculis nisi quam in libero.',
+            interactions: [
+              {
+                button: {
```

Comment:
We gebruiken in de interactions nu alleen een button, deze `button` laag is niet nodig